storagenode/trust/service.go

```diff
@@ -24,12 +25,14 @@ type Pool struct {
 	trustedSatellites  map[storj.NodeID]*satelliteInfoCache
 }
 
+// satellite info cache, caches information about a satellite.
```

Comment:
Should it be `satelliteInfoCache` to match the type name?

Reply:
yes